test/Microsoft.AspNetCore.Routing.Tests/Matchers/InstructionMatcher.cs

```diff
@@ -1,16 +1,14 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Runtime.InteropServices;
-using System.Threading.Tasks;
 using Microsoft.AspNetCore.Http;
 
 namespace Microsoft.AspNetCore.Routing.Matchers
 {
-    internal class InstructionMatcher : Matcher
+    internal class InstructionMatcher : MatcherBase
```

Comment:
What are your plans for when to remove `InstructionMatcher`?

Reply:
It might be a good idea to just remove it now. I've been realizing as I do other thing that I keep finding more places to work on it and keep it building/running/passing etc. I'll create another PR that removes it.

Reply:
Rebased on top of `InstructionMatcher`'s deletion.